src/components/Tables/Table.tsx

```diff
@@ -149,13 +149,15 @@ interface Props<Data, DbData extends Record<string, any>, PaginationType> {
             ? ClientSideFilter<Data, any>
             : ServerSideFilter<Data, any, DbData>
     >;
+    parcelRowBreakPoints?: number[]; //This is a list of indices where the line will be drawn when sorting by packing date/slot
```

Comment:
1. this is the Table component, so we shouldn't refer to parcels here, as the Table component is used for other tables, not just the parcels table.
2. We don't need this comment, the code should aim to be self-explanatory, and I think `rowBreakPoints` is a self-explanatory variable name